src/tfvc/tfvc-extension.ts

```diff
@@ -5,44 +5,58 @@
 "use strict";
 
 import { window, workspace } from "vscode";
-import { VsCodeUtils } from "../helpers/vscodeutils";
+import { ExtensionManager } from "../extensionmanager";
 import { Tfvc } from "./tfvc";
 import { Repository } from "./repository";
 import { UIHelper } from "./uihelper";
 import { IPendingChange } from "./interfaces";
 
 export class TfvcExtension  {
+    private _tfvc: Tfvc = undefined;
+    private _repo: Repository = undefined;
+    private _manager: ExtensionManager;
```

Comment:
I'm OK with not setting variables directly to undefined like you did here with ExtensionManager.  But it'd be great to be consistent (and remove them from 15 and 16).  I hope the rest of the code reflects it, but I only like to use "undefined" when I need to.